CHANGELOG.md

```diff
@@ -2,19 +2,19 @@
 
 Select a Node.js version below to view the changelog history:
 
-* [Node.js 13](doc/changelogs/CHANGELOG_V13.md) - **Current**
-* [Node.js 12](doc/changelogs/CHANGELOG_V12.md) - **Long Term Support**
-* [Node.js 11](doc/changelogs/CHANGELOG_V11.md) - End-of-Life
-* [Node.js 10](doc/changelogs/CHANGELOG_V10.md) — Long Term Support
-* [Node.js 9](doc/changelogs/CHANGELOG_V9.md) — End-of-Life
-* [Node.js 8](doc/changelogs/CHANGELOG_V8.md) — End-of-Life
-* [Node.js 7](doc/changelogs/CHANGELOG_V7.md) — End-of-Life
-* [Node.js 6](doc/changelogs/CHANGELOG_V6.md) — End-of-Life
-* [Node.js 5](doc/changelogs/CHANGELOG_V5.md) — End-of-Life
-* [Node.js 4](doc/changelogs/CHANGELOG_V4.md) — End-of-Life
-* [io.js](doc/changelogs/CHANGELOG_IOJS.md) — End-of-Life
-* [Node.js 0.12](doc/changelogs/CHANGELOG_V012.md) — End-of-Life
-* [Node.js 0.10](doc/changelogs/CHANGELOG_V010.md) — End-of-Life
+* [Node.js 13](doc/changelogs/CHANGELOG_V13.md) **Current**
```

Comment:
This change will break the version detection automation in 
https://github.com/nodejs/node/blob/de6cbd0e374e3d9209ddf56d1041b83b41efcbe6/tools/doc/versions.js#L64

Maybe it'll be reworked away in https://github.com/nodejs/node/issues/32077 but for the moment either put the dashes back in (making them consistent, if you like) or adjust the regular expression in the tooling.

Reply:
The test for this, by the way, is in `doctool` ([`test/doctool/test-doctool-versions.js`](https://github.com/nodejs/node/blob/master/test/doctool/test-doctool-versions.js)) but you'll probably need to run it with `NODE_TEST_NO_INTERNET` (https://github.com/nodejs/node/pull/31849) set so that it uses the local copy of `CHANGELOG.md` instead of pulling down the latest from `master`.

Reply:
Oh, this is good! This simplifies the regular expression.